bumpversion/__init__.py

```diff
@@ -950,6 +950,9 @@ def main(original_args=None):
     }
     vcs_context.update(time_context)
     vcs_context.update(prefixed_environ())
+    vcs_context.update({'current_' + part: current_version[part].value for part in current_version})
+    vcs_context.update({'new_' + part: new_version[part].value for part in new_version})
+    vcs_context.update({part: new_version[part].value for part in new_version})
```

Comment:
OK, let's drop the unprefixed version.

The idea was to provide consistency with `serialize`, where you can use `major`/`minor`/`patch` directly without prefix too.  But at least there it's always clear that you're referring to the 'new' part.  In `replace` it would be confusing to use a part without prefix.